scss/civicrm/administer/_administer.scss

```diff
@@ -1,3 +1,6 @@
+@import "common/mixins";
+@import "permissions/access";
```

Comment:
i don't think i agree with this, we should have only one common main file where we import all the partials. What if you need the common mixins in some other partial? are you going to import them again?